internal/ssm/errors.go

```diff
@@ -0,0 +1,117 @@
+package ssm
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	awsssm "github.com/aws/aws-sdk-go/service/ssm"
+	"os"
+)
+
+func CheckSSMError(a action, err error)  {
+	CheckBaseSSMErrors(err)
+	switch a {
+	case Get:
+		CheckSSMGetParameterError(err)
+	case Save:
+		CheckSSMPutParameterError(err)
+	}
+}
+
+func CheckRegion(err error) {
+	if awsErr, ok := err.(awserr.Error); ok {
+		switch  awsErr.Code() {
+		case "MissingRegion":
+			fmt.Println(aws.ErrMissingRegion)
+			os.Exit(1)
+		}
+	}
+}
+
+func CheckBaseSSMErrors(err error) {
```

Comment:
we should return errors instead of crashing. makes it hard to make it programmatic which you crash like this. maybe let the caller decide what to do on all these.

Reply:
Makes sense.  I will move them up to the ssm driver.